bodhi/tests/server/consumers/test_masher.py

```diff
@@ -155,6 +155,10 @@ def test___init___sets_bugtracker(self, set_bugtracker):
 
         set_bugtracker.assert_called_once_with()
 
+    @mock.patch('bodhi.server.scripts.clean_old_mashes.NUM_TO_KEEP', 2)
+    def test_clean_up(self, clean_up):
+        self.assertEquals(len(clean_up.call_args_list), 1)
```

Comment:
This test doesn't run the code and doesn't assert that the clean up call was made.